src/ui/components/achievements/components/GenericAchievementCard.tsx

```diff
@@ -87,65 +88,75 @@ Those props are provided by the GenericAchievementCard and must be passed down t
 
   return (
     <GenericCardContainer>
-      <Spacer.Flex flex={2} />
+      <Spacer.Flex flex={dimensions.height < 600 ? 1 : 2} />
       <StyledLottieContainer>
         <StyledLottieView
           key={props.activeIndex}
           ref={animationRef}
           source={props.animation}
           loop={false}
-          resizeMode="contain"
         />
       </StyledLottieContainer>
       <Spacer.Flex flex={1} />
       <StyledTitle>{props.title}</StyledTitle>
       <StyledSubTitle>{props.subTitle}</StyledSubTitle>
-      <Spacer.Flex flex={1} />
+      <Spacer.Flex flex={0.5} />
       <StyledBody>{props.text}</StyledBody>
       <Spacer.Flex flex={2} />
-      <Animatable.View ref={animatedButtonRef}>
-        {props.activeIndex === props.index ? (
-          <ButtonPrimary title={props.buttonText} onPress={props.buttonCallback} />
-        ) : (
-          <InvisibleButtonHeight testID="invisible-button-height" />
+      <BottomButtonsContainer>
+        <Animatable.View ref={animatedButtonRef}>
+          {props.activeIndex === props.index ? (
+            <ButtonPrimary title={props.buttonText} onPress={props.buttonCallback} />
+          ) : (
+            <InvisibleButtonHeight testID="invisible-button-height" />
+          )}
+        </Animatable.View>
+        {!props.lastIndex && (
+          <FlexContainer>
+            <ButtonPrimaryWhite title={_(t`Passer`)} onPress={props.skip} />
+          </FlexContainer>
         )}
-      </Animatable.View>
-      {!props.lastIndex ? (
-        <React.Fragment>
-          <Spacer.Flex flex={1} />
-          <ButtonPrimaryWhite title={_(t`Passer`)} onPress={props.skip} />
-        </React.Fragment>
-      ) : (
-        <Spacer.Flex flex={3} />
-      )}
+      </BottomButtonsContainer>
+      <Spacer.Flex flex={2} />
     </GenericCardContainer>
   )
 }
 
+const FlexContainer = styled.View({
+  flex: 1,
+  marginTop: getSpacing(4),
+})
+
+const BottomButtonsContainer = styled.View({
+  flex: 1,
+  justifyContent: 'flex-end',
+})
+
 const InvisibleButtonHeight = styled.View({
   height: getSpacing(12),
 })
 
 const StyledLottieContainer = styled.View({
   flexGrow: 1,
   alignItems: 'center',
+  justifyContent: 'center',
+  height: '30%',
 })
 
 const GenericCardContainer = styled.View({
   flex: 1,
   paddingHorizontal: getSpacing(5),
 })
 
-const StyledLottieView = styled(LottieView)({
-  width: getSpacing(60),
-  height: getSpacing(60),
+const StyledLottieView = styled(AnimatedLottieView)({
+  flexGrow: 1,
 })
 
-const StyledTitle = styled(Typo.Title1)({
+const StyledTitle = styled(dimensions.height < 600 ? Typo.Title3 : Typo.Title1)({
```

Comment:
Comme vu ensemble, ce serait bien de faire ces changements dans le fichier `typography.ts` afin d'être consistent sur toute l'app.
```suggestion
const StyledTitle = styled(Typo.Title1)({
```

Reply:
En attente des retours graphiques avant évolution.